### DISCUSSION TITLE
mcool question

### DISCUSSION THREAD

**Discussion by Zheng19w** [open2c/cooler#393] [Q&A] [answered]
Hello, I have an mcool file, but it contains X and Y sex chromosomes, as well as some scafold. Is there any way to remove them and only keep autosomes 1 to 18

![image](https://github.com/open2c/cooler/assets/106860091/64ca9e5a-afae-4bf8-b566-0aaf2866093e)

**Comment by manzt** [comment, marked as answer]
You should be able to use [cooltools rearrange](https://cooltools.readthedocs.io/en/latest/cli.html#cooltools-rearrange) to do this!
